Tests/PointFreeTests/AuthTests.swift

```diff
@@ -9,15 +9,20 @@ import XCTest
 
 class AuthTests: TestCase {
   func testAuth() {
-    let request = URLRequest(url: URL(string: "http://localhost:8080/github-auth?code=deadbeef")!)
-      |> \.allHTTPHeaderFields .~ [
-        "Authorization": "Basic " + Data("hello:world".utf8).base64EncodedString()
-    ]
+    // NB: It seems that the result of OpenSSL crypto on Linux is not deterministic, although its decryption
+    //     is, so we cannot do snapshot tests on encrypted values :/ We will still run these tests on
+    //     macOS at least.
```

Comment:
so I think maybe we just cant take snapshot tests on Linux that contain encrypted data cause it changes in the OpenSSL implementation of common crypto functions. I think maybe that's fine as long as we continue running these tests on Mac.

ok maybe not _fine_, but perhaps the best we can do...